dtos/http_request.go

```diff
@@ -0,0 +1,28 @@
+package dtos
+
+type FormCreationRequest struct {
+	Title  string `json:"title"`
+	UserId int64  `json:"userId"`
+}
+
+type FormUpdateRequest struct {
+	Title                 string `json:"title"`
+	SendEmailOnSubmission bool   `json:"sendEmailOnSubmission"`
+	UserId                int64  `json:"userId"`
+}
+
+// use for creating/updating question
+type QuestionUpdateRequest struct {
+	FormId                  int64  `json:"formId"`
+	Title                   string `json:"title"`
+	Type                    string `json:"type"`
+	Points                  int    `json:"points"`
+	IsReq                   bool   `json:"isRequired"`
+	IsPartialMarkingEnabled bool   `json:"isPartialMarkingEnabled"`
+}
+
+// use for creating/updating option
+type OptionUpdateRequest struct {
```

Comment:
- `OptionUpdateRequest` -> `UpdateOptionRequestDto`
- we will also need a option id that has to be updated? 
or should i send you all the options for a question with the required details like `title`, `order` ..etc ?